package.json

```diff
@@ -23,7 +22,7 @@
   },
   "keywords": [],
   "author": "",
-  "license": "MIT",
+  "license": "Apache-2.0",
```

Comment:
O.O nice catch, I didn't even notice that this had an Apache License in the repo but referencing the MIT license here